connectivity/FEATURE_BLE/include/ble/gap/Events.h

```diff
@@ -56,6 +56,7 @@ struct AdvertisingReportEvent {
      * @param advertisingData Advertising payload.
      */
     AdvertisingReportEvent(
+        ble_error_t status,
```

Comment:
This change the semantic of `AdvertisingReportEvent`. I don't think we need to report allocation failures to the user in the `AdvertisingReportEvent` as advertising reports are not reliable. The cordio stack will silent drop events if it doesn't have memory capacity, same for the controller.

Reply:
dropped